sdk/monitor/azure-monitor-query/src/samples/java/README.md

```diff
@@ -35,6 +35,36 @@ library.
 
 Troubleshooting steps can be found [here][SDK_README_TROUBLESHOOTING].
 
+## Running Large Log Queries using Log Analytics
+
+Due to Log Analytics [service limits][monitor_service_limits], sometimes it may
+not be possible to retrieve all the expected data in a single query. For example, the number of rows returned or the maximum size of the
+data returned may exceed the stated limits. One approach for overcoming these limits is to split the queries into multiple smaller queries
+using different time ranges.
+
+This work-around allows you to avoid the cost of exporting data to a storage account (and potentially the cost of the storage account as well). 
```

Comment:
```suggestion
This workaround allows you to avoid the cost of exporting data to a storage account (and potentially the cost of the storage account as well). 
```